zipline/pipeline/engine.py

```diff
@@ -32,6 +32,11 @@
 from zipline.utils.sharedoc import copydoc
 
 
+DATE_INDEX_NAME = 'dates'
```

Comment:
It bothers me that `dates` is plural here but `sid` is singular.  It's probably not worth breaking all the consumers of this to make the names consistent though :(.

Reply:
I'll change it to `sids`. All of the downstream consumers currently change the name of the index from (None, None), to (`dates`, `sid`).